hardhat.config.ts

```diff
@@ -61,25 +61,7 @@ const config: HardhatUserConfig = {
     compilers: [DEFAULT_CONTRACT_COMPILER_SETTINGS],
     overrides: {
       "contracts/HubPool.sol": LARGE_CONTRACT_COMPILER_SETTINGS,
-      "contracts/Arbitrum_SpokePool.sol": {
```

Comment:
For some reason, without removing this I got this error

```
These files and its dependencies cannot be compiled with your config. This can happen because they have incompatible Solidity pragmas, or don't match any of your configured Solidity compilers.

  * contracts/Linea_SpokePool.sol
  * contracts/Arbitrum_SpokePool.sol
  * contracts/SpokePoolVerifier.sol
```

However I also don't think these lines are needed since we can fix compiler versions for these three contracts. To do so I needed to change the SpokePoolVerifier's `pragma` statement